cmd/g_dto.go

```diff
@@ -0,0 +1,47 @@
+package cmd
```

Comment:
Currently no UT is written for this change, I will add UT whenever I get a chance.